odoo/cli/command.py

```diff
@@ -81,8 +109,12 @@ def main():
         # No command specified, default command used
         command_name = 'server'
 
-    if command := find_command(command_name):
-        o = command()
-        o.run(args)
-    else:
-        sys.exit('Unknown command %r' % (command,))
+    command = None
+    if Command.is_valid_name(command_name):
+        initialize_sys_path()
```

Comment:
move this line inside `find_command`, it is not necessary when you are using one of the builtin commands (99.99% of the time)